pkg/destroy/aws/aws.go

```diff
@@ -184,6 +184,51 @@ func (o *ClusterUninstaller) Run() error {
 		}
 	}
 
+	if err := o.deleteInstanceProfileHack(awsSession); err != nil {
+		o.Logger.Debug(err)
+	}
+
+	return nil
+}
+
+// THIS IS A HORRIBLE HACK! Users keep ending up with all of their instance and the roles
+// deleted, but not the instance profiles. This horrific hack guesses the name of the
+// instance profiles and delete them. It's terrible. But it works even if something else
+// deleted artifacts behind our back.
+func (o *ClusterUninstaller) deleteInstanceProfileHack(awsSession *session.Session) error {
+	o.Logger.Debug(fmt.Sprintf("%v", o))
+	clusterID := ""
```

Comment:
We use the term `clusterName` instead of `clusterID` for this elsewhere in the installer code.